src/cocaine/burlak/comm_state.py

```diff
@@ -129,6 +131,19 @@ def mark_failed(self, app, profile, state_version, tm, reason=None):
 
         self.mark_dirty()
 
+    def mark_pending_start(self, app, workers, profile, state_version, tm):
+        self.state.update(
+            {
+                app: CommittedState.Record(
+                    'SPOOLING',
```

Comment:
It is not a blocker for this PR, but could we make constants for states like `SPOOLING`, `STARTED`, `FAILED` etc. in a next one?